pom.xml

```diff
@@ -130,6 +130,31 @@
             <artifactId>spring-kafka-test</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
@chenpiaoping This is a very good start! Thanks for contributing to the project.